man/systemd.slice.xml

```diff
@@ -98,6 +87,29 @@
     </refsect2>
   </refsect1>
 
+  <refsect1>
+    <title>Options</title>
+
+    <para>See <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+    for the common options of all unit configuration files. The common configuration items are configured
+    in the generic <literal>[Unit]</literal> and <literal>[Install]</literal> sections. The
+    slice specific configuration options, such as the generic resource control settings described in
+    <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+    are configured in the <literal>[Slice]</literal> section. The options specific to the "[Slice]" section
+    of scope units are the following:</para>
+
+    <variablelist class='unit-directives'>
+      <varlistentry>
+        <term><varname>EnableOomdKill=</varname></term>
```

Comment:
I'd prefer if the name of the setting would indicate what this is for, not what the daemon is called that implements it. I am not really sure what a good name might be but something of "MannagedOOMKill", or "ControlOOM=" or so?